### ISSUE TITLE
[COGNITION]: Bug There is a character bug in country select data option ‘Reunion’ where it is ‘R�union’.

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#5846]
## Description

There is a character bug in the Country select element option ‘Reunion’ where it displays as ‘R�union’. Likely populated by a data feed. 

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a user, I want to view the country options available for my selection.

## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

1. Enter `https://staging.va.gov/profile/` in browser, and log in as user 56
2. Activate the Edit link next to the Mailing Address in Contact information
3. Activate the Country select element, and scroll to where Reunion would be
4. Verify that ‘Reunion’ displays as ‘R�union’

## Screenshots or Trace Logs

<img width="414" alt="20200213-authdx-country-select" src="https://user-images.githubusercontent.com/57469/74457678-9ad49b00-4e56-11ea-86e6-49ae29ef6d3b.png">

**Comment by Samara-Strauss** [comment]
So the problem here appears to be that there should be a dash above the "é". https://en.wikipedia.org/wiki/R%C3%A9union

@erikphansen where/how do we populate the country list in the address form in the profile? Wondering if this could be a problem elsewhere.

**Comment by erikphansen** [comment]
Wow, this is amazing. Great catch, @jenstrickland! This is the source, a common file (https://github.com/department-of-veterans-affairs/vets-website/blob/c53dec3228216b8cc58d12d44a91e2fff2ea3309/src/platform/user/profile/vet360/constants/countries.json#L1070) that is likely used in other places. It's been like this since June 2018.

**Comment by erikphansen** [comment]
I'll squash it now.

**Comment by erikphansen** [comment]
PR up https://github.com/department-of-veterans-affairs/vets-website/pull/11690

**Comment by Samara-Strauss** [comment]
@erikphansen thanks for the fast work on this one! Let me know when it's ready to be validated on staging.

**Comment by erikphansen** [comment]
@jenstrickland @Samara-Strauss this should be fixed on staging now.

**Comment by Samara-Strauss** [comment]
Looks good to me!
